documentation/modules/cruise-control/con-optimization-goals.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='con-optimization-goals-{context}']
+= Optimization goals overview
+
+To rebalance a Kafka cluster, Cruise Control uses optimization goals to generate xref:con-optimization-proposals-{context}[optimization proposals].  
+
+Optimization goals are targets for workload redistribution and resource utilization across a Kafka cluster.
+With a few exceptions, {ProductName} supports all the optimization goals developed in the Cruise Control project.
+These are as follows, in descending priority order:
+
+. Rack-awareness
+. Replica capacity
+. Capacity: Disk capacity, Network inbound capacity, Network outbound capacity
+//.. CPU capacity
+. Replica distribution
+. Potential network output
+. Resource distribution: Disk utilization distribution, Network inbound utilization distribution, Network outbound utilization distribution
+//.. CPU utilization distribution
+. Leader bytes-in rate distribution
+. Topic replica distribution
+. Leader replica distribution
+. Preferred leader election
+//. Intra-broker disk capacity
+//. Intra-broker disk usage distribution   
```

Comment:
what are this line witn "//" ? this list is not clear to me. Are there titles and topics?

Reply:
`//` is comment notation in Asciidoc. These goals are not supported as confirmed by @kyguy. I decided to comment out rather than delete.